Frontend/library/src/PeerConnectionController/AggregatedStats.ts

```diff
@@ -312,6 +319,14 @@ export class AggregatedStats {
      * @returns The candidate pair that is currently receiving data
      */
     public getActiveCandidatePair(): CandidatePairStats | null {
-        return this.candidatePairs.find((candidatePair) => candidatePair.bytesReceived > 0, null)
+        
+        // Check if the RTCTransport stat is not undefined
+        if (this.transport){
+            // Return the candidate pair that matches the transport candidate pair id
+            return this.candidatePairs.find((candidatePair) => candidatePair.id = this.transport.selectedCandidatePairId, null);
```

Comment:
After squinting at my screen and reading this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#callbackfn

I _think_ you what you want here is:

`return this.candidatePairs.find((candidatePair) => candidatePair.id === this.transport.selectedCandidatePairId, null);`

Reply:
Note doing a comparison instead of an assignment.